bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gems/CanopusWfs.java

```diff
@@ -183,9 +182,9 @@ private static Area offsetIntersection(final ObsContext ctx, final Set<Offset> o
             xform.translate(-p, -q);
 
             return patrolField.getArea().createTransformedArea(xform);
-        }).reduce(patrolField.getArea(), (a1, a2) -> {
+        }).reduce((a1, a2) -> {
             a1.intersect(a2);
             return a1;
```

Comment:
I'm trying to puzzle out how much to worry about the fact that the combining operation actually mutates one of the parameters.  I think I'd feel better if it were, say,

```java
final Area result = a1.clone();
result.intersect(a2);
return result;
```

Reply:
Interesting. I was somehow thinking that `a1.intersect` would return another instance

Reply:
👍 
Yes, this seems like a better idea.

Reply:
How I wish that Java `Area` and `AffineTransform` objects returned new instances when combined...